tools/depends/native/Toolchain-Native.cmake.in

```diff
@@ -51,7 +51,7 @@ set(CMAKE_EXE_LINKER_FLAGS "@host_includes@ -L@prefix@/@tool_dir@/lib")
 set(CMAKE_FIND_ROOT_PATH @prefix@/@tool_dir@)
 set(CMAKE_LIBRARY_PATH @prefix@/@tool_dir@/lib)
 if(NOT "@use_toolchain@" STREQUAL "")
-  list(APPEND CMAKE_FIND_ROOT_PATH @use_toolchain@ @use_toolchain@/@use_host@ @use_toolchain@/@use_host@/sysroot @use_toolchain@/@use_host@/sysroot/usr @use_toolchain@/@use_host@/libc @use_toolchain@/lib/@use_host@/sysroot @use_toolchain@/usr @use_toolchain@/sysroot/usr)
+  list(APPEND CMAKE_FIND_ROOT_PATH @use_toolchain@ @use_toolchain@/lib/@use_host@/sysroot @use_toolchain@/usr @use_toolchain@/sysroot/usr)
```

Comment:
Should anything that use @use_host@ be removed entirely? eg @use_toolchain@/lib/@use_host@/sysroot as well?

Reply:
Well I think so, but these folders don't exist in the webOS toolchain so they don't harm

Reply:
Let's see first if this breaks any other platform but I think also the line below is not needed as `CMAKE_FIND_ROOT_PATH` should already account for it.